ClinGen/clincoded
# Connects to #1182. GCI help doc links.

**Technical notes:**
1. A symlink has been added for the new GCI help doc.
2. The GCI help doc in this PR is a temporary/dummy copy. It needs to be replaced with the finalized version when ready.

**Steps to test:**
1. Without logging in, you should expect to see the "Help" button in the top nav bar:
![image](https://cloud.githubusercontent.com/assets/6956310/21744462/0bad210c-d4cb-11e6-9b3e-5d7241bb725a.png)

2. Click on the "Help" button and expect to see a dropdown menu consisting of 2 menu item links -- "Gene Curation" and "Variant Curation":
![image](https://cloud.githubusercontent.com/assets/6956310/21744484/5634d4fe-d4cb-11e6-8558-ee9a2d3531d5.png)

3. Clicking on either one of the menu item links should open the PDF docs in a new window/tab.

4. After logging in, you should expect to see the GCI help doc link via the "?" icon on the Dashboard page:
![image](https://cloud.githubusercontent.com/assets/6956310/21744500/b1942d22-d4cb-11e6-87da-300fc95ed13a.png)

5. Clicking on GCI help doc link should open the PDF doc in a new window/tab.

6. Also expect to see the "Help" button in the top nav bar to continue to behave the same as described above:
![image](https://cloud.githubusercontent.com/assets/6956310/21744519/159041a8-d4cc-11e6-86d9-e3c7867fe152.png)
